solarwinds_apm/apm_constants.py

```diff
@@ -35,6 +35,20 @@
     INTL_SWO_BAGGAGE_PROPAGATOR,
     INTL_SWO_PROPAGATOR,
 ]
+INTL_SWO_DEFAULT_RESOURCE_DETECTORS = [
+    "process",
+    "os",
+    "host",
+    "aws_ec2",
+    "aws_ecs",
+    "aws_eks",
```

Comment:
These should give us `container.id` 👍

Reply:
Depending on the cgroup version, the `container.id` is actually not 100% guaranteed.